### PR TITLE
Fix autoscale of linechart when data have holes.

### PR DESCRIPTION
**Pull request by PierrVanRoy**
The yScale.auto option is not properly computed when the data have holes and is positive. 

This PR filters those hole from the values of the chart so tat the min is not always 0 in that case.

Before:
image

After:
image

Reported also by: https://github.com/plouc/nivo/issues/859

### CODE REVIEW

File: packages/scales/src/compute.ts [hunk 1]
```diff
@@ -195,7 +195,7 @@ export const generateSeriesAxis = <Axis extends ScaleAxis, Value extends ScaleVa
 
     switch (scaleSpec.type) {
         case 'linear': {
-            const all = sortBy(uniq(values as number[]), v => v)
+            const all = sortBy(uniq(values as number[]).filter(v => v !== null), v => v)
```
Comment (plouc) [inline packages/scales/src/compute.ts:198]: I think we should add a unit tests for this, also maybe a comment explaining why we filter out null values, what do you think?

**Review by plouc — APPROVED** [review]

**Comment by plouc** [comment]
@PierrVanRoy, thank you for this fix, nice UI btw :) Is that a public project?

**Review by plouc — APPROVED** [review]
LGTM

**Comment by PierrVanRoy** [comment]
@plouc , we are using Nivo quite a lot at [Pigment](https://www.gopigment.com/) but sadly I have no public access to share with you :-( .
BTW, thanks a lot for this project. It's a real pleasure to use it!

**Comment by PierrVanRoy** [comment]
hello @plouc,

Do you have any visibility on when this fix will be include a nivo release? 🙏 
Just want to know if we should patch on our side or if we ca wait for an official release. 

Thanks 

Pierre

**Comment by plouc** [comment]
@PierrVanRoy, unfortunately, I'm pretty busy with work and personal matters at the moment, so it's unlikely that I'll release the fix soon.
